docs/job_tutorial.md

```diff
@@ -118,8 +118,8 @@ Below please find the detailed explanation for each of the parameters in the con
 | `taskRole.portType.beginAt`      | Integer, required          | The port to begin with in the port type, 0 for random selection |
 | `taskRole.portType.portNumber`   | Integer, required          | Number of ports for the specific type    |
 | `taskRole.command`               | String, required           | Executable command for tasks in the task role, can not be empty |
-| `taskRole.minFailedTaskCount`    | Integer, optional          | Number of failed tasks to kill the entire job, null or no less than 1, refer to [frameworklauncher usermanual](../subprojects/frameworklauncher/yarn/doc/USERMANUAL.md#ApplicationCompletionPolicy) for details |
-| `taskRole.minSucceededTaskCount` | Integer, optional          | Number of succeeded tasks to kill the entire job, null or no less than 1, refer to [frameworklauncher usermanual](../subprojects/frameworklauncher/yarn/doc/USERMANUAL.md#ApplicationCompletionPolicy) for details |
+| `taskRole.minFailedTaskCount`    | Integer, optional          | Number of failed tasks to kill the entire job, null or no less than 1, if set to null means the job will always succeed regardless any task failure. Please refer to [frameworklauncher usermanual](../subprojects/frameworklauncher/yarn/doc/USERMANUAL.md#ApplicationCompletionPolicy) for details |
+| `taskRole.minSucceededTaskCount` | Integer, optional          | Number of succeeded tasks to kill the entire job, null or no less than 1, if set to null means all tasks need to run successfully. Please refer to [frameworklauncher usermanual](../subprojects/frameworklauncher/yarn/doc/USERMANUAL.md#ApplicationCompletionPolicy) for details |
```

Comment:
if set to null means the job will only succeed until all tasks are completed and minFailedTaskCount is not triggered.